phalcon/db/adapter/pdo.zep

```diff
@@ -51,6 +51,11 @@ abstract class Pdo extends Adapter
 	 */
 	protected _pdo;
 
+	/**
+	 *
+	 */
+	abstract protected function getDsnDefaults() -> array;
```

Comment:
Provide please documentation string

Reply:
I think that the [Oracle Incubator](https://github.com/phalcon/incubator/blob/master/Library/Phalcon/Db/Adapter/Pdo/Oracle.php) will need to be updated shortly after this is merged.  So you'd need to start a 4.x incubator branch.